Articles/Blog/HelloHeatEquation.md

```diff
@@ -1,143 +1,99 @@
-
-Suppose you have a thin metal rod along the x-axis starting at position 0 and ending at position _L_. The goal is to model the temperature distribution throughout the rod at a given time, with the following (ideal) assumptions:
+#### Contributed by [Heather M. Switzer](https://github.com/heatherms27 "Heather Switzer GitHub Profile"), [Elsa Gonsiorowski](https://github.com/gonsie "Elsa Gonsiorowski GitHub Profile") and [Mark C Miller](https://github.com/markcmiller86 "Mark C Miller GitHub Profile")
 
-* An initial temperature distribution is given
-* Boundary conditions (established tempuratures at each end of the rod) are known
-* Heat only moves horizontally through the bar
-* There are no external heating/cooling sources
+["Hello World!"](https://www.thesoftwareguild.com/blog/the-history-of-hello-world/)
+is the canonical *first* program beginners write to learn new programming methodologies.
+In computational science and engineering ([CSE](https://en.wikipedia.org/wiki/Computational_engineering))
+circles, programming methodologies can involve a variety of languages, parallel execution
+and programming models, discretization techniques, interdependent libraries and more.
+An application solving the [heat equation](https://en.wikipedia.org/wiki/Heat_equation)
+represents the perfect "Hello World!" program for the CSE world.
 
-<div style="text-align:center"><img width="75%" src="https://github.com/betterscientificsoftware/images/blob/master/Blog_0719_HeatEqnBar.png" /></div>
-
-To model the temperature distribution of the rod, we can use the 1-dimensional Heat Equation given by the formula:
+## What is the Heat Equation?
+The Heat Equation, a special case of the Diffusion Equation, was developed by Joseph
+Fourier in 1822. It describes how the distribution of heat varies over space and time
+in a material. While there are many adaptations for different scenarios, beginner CSE
+programmers often start with the one-dimensional, homogeneous Heat Equation as an
+introductory problem.
 
 _&part;u / &part;t = &alpha;&nabla; &sup2;u_
 
 Where:
-* _u_ = Temperature at a position, _x_
+* _u_ = Temperature
 * _t_ = time
-* _&alpha;_  = thermal diffusity of the material
-* _&nabla;_ = The LaPlace operator
+* _&alpha;_  = thermal diffusivity of the material
+* _&nabla;_ = The Laplace operator
 
-### Assumptions and Discretization
-To make the problem more approachable, we make some simplifying assumptions:
-* _&alpha;_ is constant
-* The heat/cooling source only comes from initial/boundary conditions
+To make the equation even more approachable, we make some simplifying assumptions:
+* _&alpha;_ is constant (the material is homogeneous)
+* Apart from initial/boundary conditions, there are no other heat sources
 * Only Cartesian coordinates in 1 dimension are used
 
-Since the heat equation takes the form of a PDE over a continuous space, most programs create a discretization of the formula over a finite set of sampling points.
```

Comment:
please move the deleted text to an explanation page in the [hello-heat-equation repository](https://github.com/betterscientificsoftware/hello-heat-equation)

Reply:
Done. See [this PR](https://github.com/betterscientificsoftware/hello-heat-equation/pull/3)